src/JSON/API.dfy

```diff
@@ -0,0 +1,37 @@
+include "Serializer.dfy"
+include "Deserializer.dfy"
+include "ZeroCopy/API.dfy"
+
+module {:options "-functionSyntax:4"} JSON.API {
```

Comment:
I cannot use just `JSON`, unfortunately :( That's because definitions in the JSON module cannot use definitions in submodules of it, like `JSON.XYZ`.

I considered having two top-level modules JSON and JSONImpl, but that's not great either.

Finally, this respects the link between module names and file names (`JSON.API` in `JSON/Api`).

We ought to improve this in the new module system.

Reply:
Keeping JSON.API, it would be possible to create a file in the folder above JSON.dfy that refines this module, so that users don't need to write JSON.API? Could you do this? I don't see myself writing JSON.API to invoke JSON routines. JavaScript offers JSON.stringify and JSON.parse.

Reply:
That would introduce a mismatch between the file name and the file path.  You can, however, do `import JSON.API as JS`, which is good, I think?

Reply:
It is good, but not good enough for what we want to be a default library. I'll take a pass on that later.